feat:[CDS-96727]: review comments for jira kb for custom fields (#6847)
harness/developer-hub: kb/continuous-delivery/articles/create-cascading-fields-jira.md

---

This article talks about using the HTTP step to set the fields that are not supported by Harness using an example. 

> By default, Harness don't support cascading lists and other custom field types. For the list of supported Jira fields, go to [Add issue fields](/docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages/#add-issue-fields).

Reviewer: ```suggestion
By default, Harness doesn't support cascading lists and other custom field types. For the list of supported Jira fields, go to [Add issue fields](/docs/continuous-delivery/x-platform-cd-features/cd-steps/ticketing-systems/create-jira-issues-in-cd-stages/#add-issue-fields).
```

> We will discuss how to set a [cascading list](https://confluence.atlassian.com/jirakb/creating-dependent-cascading-lists-in-jira-server-datacenter-1142426572.html) using the [HTTP step](/docs/continuous-delivery/x-platform-cd-features/cd-steps/utilities/http-step/). This unblocks integration to jira projects which have such fields marked as mandatory.

Reviewer: ```suggestion
We will discuss how to set a [cascading list](https://confluence.atlassian.com/jirakb/creating-dependent-cascading-lists-in-jira-server-datacenter-1142426572.html) using the [HTTP step](/docs/continuous-delivery/x-platform-cd-features/cd-steps/utilities/http-step/). This unblocks integration with Jira projects that mark such fields as mandatory.
```

- URL: Enter your Jira server details. For example, `https://<your_JIRA_domain>/rest/api/2/issue`.
- Method: Select **POST** HTTP method.

> - Request Body: Enter the cURL command to create a task in project `TES` with a summary and cascading field with id `customfield_10500`. Here's a sample:  

Reviewer: ```suggestion
- Request Body: Enter the cURL command to create a task in project `TES` with a summary and cascading field with ID `customfield_10500`. Here's a sample:  
```
